### PR TITLE
[B] Fix resource pagination losing filters

### LINKED ISSUE #288: Changing pages in resources list breaks filters
Filters are reset when the page is changed project resource lists.

### PR DESCRIPTION
**Pull request by SMaxOwok**
[Fixes #288]

### CODE REVIEW

File: client/src/components/frontend/Project/Resources.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ export default class ProjectResources extends Component {
     project: PropTypes.object,
     resources: PropTypes.array,
     pagination: PropTypes.object,
+    paginationClickHandler: PropTypes.object,
```
Comment (zdavis) [inline client/src/components/frontend/Project/Resources.js]: Ok.... really, this should be an object?

File: client/src/containers/frontend/ProjectResources.js [hunk 2]
```diff
@@ -109,6 +128,7 @@ class ProjectResourcesContainer extends Component {
             project={project}
             resources={this.props.resources}
             pagination={this.props.meta.pagination}
+            paginationClickHandler={this.pageChangeHandlerCreator}
```
Comment (zdavis) [inline client/src/containers/frontend/ProjectResources.js]: You're passing a function in a prop that's supposed to be an object, per your proptype validation above.

**Comment by zdavis** [comment]
Looks like there's also a `bind` error in here somewhere.

screen shot 2017-04-25 at 8 25 11 am

**Comment by zdavis** [comment]
@SMaxOwok when you rebase this, please keep 1c2763c as a stand-alone commit. I'm a little bit concerned that changing the behavior of the meta entityUtil method will cause some issues for paginated components — please take a quick pass through the backend and make sure that events, resources, projects, etc. all work with this change.

**Comment by zdavis** [comment]
Max, this branch still does not work for me. When I go to the all resources view, set a type filter, then go to the second page, I see results from the first page (after a brief flash of the page 2 results). This behavior is not present in master, so it appears to be introduced by your branch.

**Comment by SMaxOwok** [comment]
Per our Slack discussion:  The bug described here is not a result of this PR, but rather a result of re-fetching data on location changes.  In this case, the pagination utility appends `#pagination-target` to the URL, which causes a refetch in the component.

See #304

**Pull request merged**